pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/text/TextIndexType.java

```diff
@@ -70,6 +71,11 @@ public class TextIndexType extends AbstractIndexType<TextIndexConfig, TextIndexR
   protected static final Logger LOGGER = LoggerFactory.getLogger(TextIndexType.class);
 
   public static final String INDEX_DISPLAY_NAME = "text";
+  // TODO: Should V1Constants.Indexes.LUCENE_TEXT_INDEX_DOCID_MAPPING_FILE_EXTENSION be added here?
```

Comment:
@atris Can you bring some light here?

Reply:
I think it should be placed together in code since after all it is for one of the on-disk structures we maintain in Lucene text index

Reply:
> It is a mapping file we maintain to map Lucene internal docID to pinot segment docID as the former is not guaranteed to be the same as our representation.

That means that a text file may require to have not one but two files? If that is the case, we would need to adapt the code a bit more.